### ISSUE TITLE
show server info

### ISSUE THREAD

**Issue by laredson** [kldzj/pzmod#9]
It is kinda done by the "Update server info" while it let you change...
But it would be cool to have an option to just view the current server info as it will be shown in the ingame screen

**Comment by kldzj** [comment]
Good idea. Also these are just the most basic settings for now, do you have any other very important config keys in mind?

**Comment by laredson** [comment]
to set PVP on and off?

sometimes I edit the line for starting gear to give something to players, a gun or weapon usually

do you mean that kind of config?

**Comment by laredson** [comment]
To be shown they must be things that you're modified compared to a "standard" vanilla server.ini

Also the lines for anticheat are usually changed for some mods that will crash in a MP server without certain anticheat disabled
If you want to add an option to that, I don't think is necessary as we are used to change it on edit manually, but to be shown if it's different than vanilla. That is only show "false" anticheats, since vanilla brings all "true" as default

**Comment by kldzj** [comment]
> do you mean that kind of config?

Exactly :)

**Comment by laredson** [comment]
For anticheat:

If API can read mod's description, try to catch "anticheat" and/or "anti-cheat" to show a little reminder for that mod showing the line or a button "show mod description". 
Maybe there's another way to consider one mod is asking to turn off some anticheat... 

A list for anticheat with the [  ] and [x] would be useful
